templates/persistence/graphdb-master-pvc.yaml

```diff
@@ -7,7 +7,9 @@ metadata:
   labels:
     name: graphdb-master-{{ $master_index }}-data-pvc
 spec:
-  volumeName:  {{ $.Values.graphdb.masters.persistence.volumeNamePrefix }}-master-{{$master_index}}-pv
+  {{- if eq $.Values.graphdb.masters.persistence.storageClassName "standard" }}
```

Comment:
Do NOT base the volume name on the class name. Check if volumeNamePrefix is defined.